src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Unix.cs

```diff
@@ -9,11 +9,8 @@ internal sealed partial class CultureData
     {
         private bool InitCultureDataCore() => InitIcuCultureDataCore();
 
-        private void InitUserOverride(bool useUserOverride)
-        {
-            // Unix doesn't support user overrides
-            _bUseOverrides = false;
```

Comment:
Can/should the `_bUseOverrides` field be moved to the Windows file, since it is only used by the Windows code?

Reply:
It could but it's quite a lot of new redirections and changes all over the place. I think it can wait for another PR